packages/website/versioned_docs/version-latest/release-notes/release-2022-12-07.md

```diff
@@ -14,7 +14,7 @@ This release contains **breaking changes**
 
 With the decorator signature `extern dec` added in previous release and the doc comment support added in this version the IDE is able to provide a signature help for decorators
 
-![](./signature-help.png)
+![signature-help](./signature-help.png)
```

Comment:
```suggestion
![Illustration showing Signature help in VSCode](./signature-help.png)
```

Reply:
I think this one must also have the same file in the `docs/` folder, thjis change will be overritten at the next release otherwise.